src/operator/api/v1alpha3/clientintents_types.go

```diff
@@ -222,9 +222,12 @@ func (in *ClientIntents) GetFilteredCallsList(intentTypes ...IntentType) []Inten
 }
 
 func (in *ClientIntents) GetIntentsLabelMapping(requestNamespace string) map[string]string {
-	otterizeAccessLabels := map[string]string{}
+	otterizeAccessLabels := make(map[string]string)
 
-	for _, intent := range in.GetFilteredCallsList(IntentTypeHTTP, IntentTypeKafka) {
+	for _, intent := range in.GetCallsList() {
```

Comment:
why not use GetFilteredCallsList?

Reply:
Because that filters in, and we need a filter out - we also wanna include intents with no type, or this breaks network policies (which require no type)